README.txt

```diff
@@ -1,7 +1,32 @@
 Socrata module - http://drupal.org/project/socrata
 ==================================================
-Provides an integration point for Socrata within Drupal via their SODA2 interface, including a Views
-query solution and an input filter.
+Provides an integration point for Socrata within Drupal via their SODA2 interface, including a Views query solution and an input filter.
+
+There are four modules in this project:
+
+Socrata
+-------
+Manages the dataset instances and allows for import/export of these definitions. When enabled, you will find a menu item under Structure > Socrata that will allow you to manage those. They can then be used in conjunction with other modules.
```

Comment:
One little quibble, instead of "instances", how about we call them "endpoints" as we do within the UI?  Then we introduce the user to a term used throughout the Socrata experience.